[RFC] vim-patch:7.4.729

jbradaric opened the pull request:
Problem:    Occasional crash with 'list' set.
Solution:   Fix off-by-one error. (Christian Brabandt)

jbradaric committed:
vim-patch:7.4.729

Problem:    Occasional crash with 'list' set.
Solution:   Fix off-by-one error. (Christian Brabandt)

sethjackson commented on src/nvim/screen.c:
```diff
@@ -3469,7 +3469,7 @@ win_line (
                                 && (colnr_T)vcol == wp->w_virtcol)))
-                   && lcs_eol_one >= 0) {
+                   && lcs_eol_one > 0) {
           /* Display a '$' after the line or highlight an extra
```
Lint is failing because of the comment style. These need to be changed to `//`.

You can run `make lint` locally to check if you've fixed all the linter errors.

Commit:
screen: Fix linting errors

Commit:
Add eol to listchars for mouse specification tests

This is consistent with VIM behavior. When showing a visual selection,
VIM only extends it past the last character if eol is in listchars (even
if nolist is set).
